Movies Time/ViewController/MovieDetailsViewController.swift

```diff
@@ -197,14 +197,14 @@ class MoviesDetailsViewController: UIViewController {
         overviewText.text = movie.overview
     }
     
-    private func viewsFor() -> UIView{
+    private func viewsProperties() -> UIView{
         let view = UIView()
         view.translatesAutoresizingMaskIntoConstraints = false
         view.backgroundColor = Constants.color
         return view
     }
     
-    private func labelFor() -> UILabel{
+    private func labelProperties() -> UILabel{
```

Comment:
same comment here, it would be more readable if the name was "createGenericLabel" or "createReusableLabel"